### PR TITLE
Newsletter 7: Add updates about Grumpy Visitors

### PR DESCRIPTION
**Pull request by mvlabat**
I'm not sure the markdown is correct... I couldn't find the way to preview this post.

I'd be grateful if anyone could confirm it's good. :)

------

_Part of #64 ("Newsletter 7: Coordination/Tracking")_

### CODE REVIEW

File: content/posts/newsletter-007/index.md [hunk 1]
```diff
@@ -527,6 +527,19 @@ February's full weekly devlogs: "This Week In Veloren...":
 
 [veloren]: https://veloren.net
 
+### [Grumpy Visitors first update. Winter progress][grumpy_visitors]
+![Magic missiles in Grumpy Visitors](grumpy_missiles.jpg)
+
```
Comment (ozkriff) [inline content/posts/newsletter-007/index.md]: Could you add a short description of the game, please?

**Comment by 17cupsofcoffee** [comment]
For future reference - you can preview the post either by:

* If you have Zola installed, run `zola serve --drafts` in the root of the project.
* If you don't want to install anything, push your branch to your fork on GitHub and [open the file in your browser](https://github.com/mvlabat/rust-gamedev.github.io/blob/grumpy-visitors-winter-update/content/posts/newsletter-007/index.md).

The latter doesn't include the site's styling, but it's usually fine for quick previews 😄

**Comment by mvlabat** [comment]
@17cupsofcoffee, thank you! Didn't know about the `--drafts` flag, that's helpful :)

**Comment by ozkriff** [comment]
:timer_clock: Merging in a few hours if there no objections.

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**